WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -149,6 +150,24 @@ protected void onNewIntent(Intent intent) {
     private void launchWithNoteId() {
         if (isFinishing() || getIntent() == null) return;
 
+        // Check for push authorization request
+        if (getIntent().hasExtra(NotificationsUtils.ARG_PUSH_AUTH_TOKEN)) {
+            Bundle extras = getIntent().getExtras();
+            String token = extras.getString(NotificationsUtils.ARG_PUSH_AUTH_TOKEN, "");
+            String title = extras.getString(NotificationsUtils.ARG_PUSH_AUTH_TITLE, "");
+            String message = extras.getString(NotificationsUtils.ARG_PUSH_AUTH_MESSAGE, "");
+            long expires = extras.getLong(NotificationsUtils.ARG_PUSH_AUTH_EXPIRES, 0);
```

Comment:
Good catch, it should still show the prompt if we don't receive an expiry date. Fixed in 577b9bcf44d73012978b96f888f297b37f6c042c